.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+ version: 2.1
+
+ orbs:
+  win: circleci/windows@2.2.0
```

Comment:
Any reason for windows?

Reply:
We used pregenerated template for .NET that CircleCI provides to initialize the script file. As we investigate more we can change it.
Do you have suggestions on which orb we should use?

Reply:
Ah, OK. We're using an image instead of an orb: https://hub.docker.com/_/microsoft-dotnet-core-sdk:
```
docker:
      - image: mcr.microsoft.com/dotnet/core/sdk:3.1
```

But I can't actually see what that's built on, and I don't know what the windows orb does, so can't really comment about which is better :)